minigui/layer.ts

```diff
@@ -475,10 +488,12 @@ class Annotations extends Layer {
     let position = this.board.position;
     this.annotations.clear();
     for (let annotation of position.annotations) {
-      let byShape = this.annotations.get(annotation.shape);
+      // this isn't really working with the enums, but its not clear how the
+      // annotations on a position should be grouped, otherwise.
+      let byShape = this.annotations.get(annotation.label);
```

Comment:
We don't actually need this grouping any more if we're just rendering text. Previously, the idea was that we'd group all annotations by type (circle, triangle, etc) first, then render them in batches.

I _think_ you can remove all this and just do `this.annotations = position.annotations`

Reply:
Done.